src/Composer/ScriptHandler.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace ApiPlatform\Core\Composer;
+
+use Composer\Script\Event;
+use Symfony\Component\Filesystem\Filesystem;
+use Symfony\Component\Finder\Finder;
+
+class ScriptHandler
+{
+    const SWAGGER_UI_DESTINATION = '/web/swagger-ui';
```

Comment:
Hmm, to me it's cleaner than adding `DIRECTORY_SEPARATOR` below:

`$cwd.DIRECTORY_SEPARATOR.self::SWAGGER_UI_SOURCE`